src/main/java/com/chariotsolutions/meetupnotifier/meetup/api/MeetupQuery.java

```diff
@@ -12,16 +11,15 @@
 
 import java.io.IOException;
 
-public class App {
-
+public class MeetupQuery {
   /**
-   * Currently being used for testing purposes.
-   * @param args Commandline arguments
-   * @throws IOException In the event of a failed meetups.com API call.
+   *
+   * @return ResultsObject containing the results of the call.
+   * @throws IOException In case of an error communicating with the API.
    */
-  public static void main(String[] args) throws IOException {
+  public static Results fetchMeetups() throws IOException {
     CloseableHttpClient client = HttpClientBuilder.create().build();
-    HttpGet getRequest = new HttpGet("https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_urlname=phillypug&photo-host=public&page=20&fields=timezone&order=time&desc=false&status=upcoming&sig_id=188504148&sig=3e8db7cb314f7e2b25f0cc7054c5dea549483eb2");
+    HttpGet getRequest = new HttpGet("https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_urlname=phillypug&text_format=plain&photo-host=public&page=20&fields=timezone&order=time&desc=false&status=upcoming&sig_id=188504148&sig=87e57bf49005cb980d48f87bd8c4bc53df8fa1f3");
```

Comment:
- The `MeetupQuery` class should expose parameters that define the query for clients (the meetup group, text format, etc.).  This way different queries can be built using this class.  
- The operation of fetching the results should be extracted  into a separate class that can take a `MeetupQuery` and execute it.  This separates the query from the execution of it.  
- Then upon startup if you end up reading the list of groups to check from a properties file, etc. you can build up `MeetupQuery` instances which will then be executed.

You probably were taking the agile approach for now and planning on doing this later anyway, but it won't hurt to do it now.